blazar/additions/blazar-manager/hooks/on_before_end.py

```diff
@@ -145,14 +162,23 @@ def main(argv):
     html = render_template(vars=template_vars)
 
     # read email host from blazar.conf
-    email_host = "127.0.0.1"
+    email_host = os.environ.get("SMTP_HOST", "127.0.0.1")
+    # smtplib's default is 0, defaults to OS implementation
+    email_port = os.environ.get("SMTP_PORT", "0")
+    email_tls = os.environ.get("SMTP_TLS")
+    email_user = os.environ.get("SMTP_USER")
+    email_password = os.environ.get("SMTP_PASSWORD")
     blazar_config = configparser.ConfigParser()
     try:
         blazar_config.read("/etc/blazar/blazar.conf")
         email_host = blazar_config["physical:host"]["email_relay"]
+        email_port = blazar_config["physical:host"].get("email_port", 0)
+        email_tls = blazar_config["physical:host"].get("email_tls")
+        email_user = blazar_config["physical:host"].get("email_user")
+        email_password = blazar_config["physical:host"].get("email_password")
```

Comment:
why are we setting these twice?

AFAIK dict.get(key) will return Null if the key isn't in the dict, so it would always overwrite the env vars?
Do we want the env vars or the config file to take precedence?

Reply:
Good point. I guess this depends on what we want to take priority, I was testing with environment variables for simplicity and thought it was be good to have them. I was expecting this config section would raise an exception at line 174 in most cases. Maybe we don't want to support env vars at all.